### ISSUE TITLE
NVActivityIndicatorViewable doesn't work on multiple VCs or calls 

### ISSUE THREAD

**Issue by amolpokale1** [ninjaprox/NVActivityIndicatorView#161]
when we show the activity indicator and perform any ui changes on view then activity indicator get disappear

**Comment by ninjaprox** [comment]
@amolpokale1: Could you elaborate on the issue and use case? Either screenshots, gifs, snippets would be helpful.

**Comment by amolpokale1** [comment]
when I land on my screen I request web_service_call_1 and show the activity indicator, On success of web_service_call_1  response I added new view on my UIView and request web_service_call_2, when I get the response from web_service_call_2 then I hide the activity indicator

**Expected** - activity indicator should display for my both request
 
**Reality** -  activity indicator not displayed some time

**Comment by invinciible** [comment]
I'm using delegate methods of NVActivityIndicatorViewable, I found a issue while using this in multiple ViewController
I have this in parentView controller and it worked fine, but in the child VC not working and is I call that child VC from someOther parent VC who's not conforming to this ,than it works.
weird.

**Comment by marconascimento** [comment]
I'm facing exactly the same issue.
Any solution so far?

**Comment by invinciible** [comment]
Mine fix was to stop the animation before starting again.

**Comment by rehannali** [comment]
Stop activity indicator while starting new one. 
There is another issue is you are creating multiple instance of activity indicator and never remove it before using it again.

**Comment by ninjaprox** [comment]
@rehannali: Could you create a new issue for the other issue you mentioned above?

**Comment by ninjaprox** [comment]
Clean up stalled issues.
